### PR TITLE
ZEN-14030: Add date-picker

### PR DESCRIPTION
**Pull request by sagaban**
[Jira ticket](https://reciprocitylabs.atlassian.net/browse/ZEN-14030)

### CODE REVIEW

**Comment by zenbot-0** [comment]
🔬 [Click here](https://reciprocity.github.io/zenlib/branches/feature/implement-datepicker) to test your changes on Storybook

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # ZenLib is a Vue.js components library
+
```
Comment (leferreyra) [inline README.md]: 👀

File: README.md [hunk 2]
```diff
@@ -64,6 +65,23 @@ If you want to generate a coverage report for the unit tests, there's an `npm ru
 
 We run coverage checks on each PR so try not to ship code without tests ;)
 
+### Adding a new package/component
+
+1. Create a folder under the `package` folder. Name it as the component, using Kebab case. It should contain an `index.js` file that export the main component.
```
Comment (leferreyra) [inline README.md]:
```suggestion
1. Create a folder under the `packages` folder. Name it as the component, using Kebab case. It should contain an `index.js` file that exports the main component.
```

File: README.md [hunk 3]
```diff
@@ -64,6 +65,23 @@ If you want to generate a coverage report for the unit tests, there's an `npm ru
 
 We run coverage checks on each PR so try not to ship code without tests ;)
 
+### Adding a new package/component
```
Comment (leferreyra) [inline README.md]: 💯

File: packages/date-picker/README.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    time: null
+  },
+  components: {
+    DatePicker
+  }
+};
+</script>
+```
+
+### Props
```
Comment (leferreyra) [inline packages/date-picker/README.md]: 👏

File: README.md (hunk 1 above)
Comment (sagaban) [reply README.md]: My IDE autoformat...

**Pull request merged**